app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    string_list = command.split(" ")
+    if len(string_list) == 3 and string_list[0] == "cp":
+        if string_list[1] != string_list[2]:
+            with open(string_list[1], "r") as file_in, \
+                    open(string_list[2], "w") as file_out:
```

Comment:
what are string_list[0], string_list[1] and string_list[2]?
create separated variables with declarative names
-------------------
```
with (open(string_list[1], "r") as file_in,
      open(string_list[2], "w") as file_out):
```

Reply:
Hello, v2